CI/travis.linux.after_success.sh

```diff
@@ -40,7 +40,7 @@ if { [ "${DEPLOY}" = "deploy" ]; } ||
   COMMIT_DATE=$(git show -s --format="%cs" | tr -d '-')
   YESTERDAY_DATE=$(date -d "yesterday" '+%F' | tr -d '-')
 
-  git clone https://github.com/Mudlet/installers.git "${BUILD_DIR}/../installers"
+  git clone https://github.com/Mudlet/installers.git -b improve-linux-packaging "${BUILD_DIR}/../installers"
```

Comment:
Is it already installed by default in macOs builds - or do you need to adjust the `./CI/travis.macos.after_success.sh` as well?

Reply:
macOS does well out of the box here. Qt pays more attention there.